charts/argo-rollouts/values.yaml

```diff
@@ -229,6 +229,8 @@ serviceAccount:
   # -- The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
   name: ""
+  # -- Specifies wether a service account should be automounted
+  automount: true
```

Comment:
Regarding the comment and default value, please align to https://github.com/argoproj/argo-helm/pull/2625 .
Reason of the default value `false` is also mentioned in the PR.

Reply:
@yu-croco The comments and default values have been updated. Thank you for the feedback.

Reply:
@yu-croco looking at the referenced PR again, the default value is set to `true`. 

You can also see in this in `main` https://github.com/argoproj/argo-helm/blob/936101865d3090183fc4979546685ece7f78ac23/charts/argo-cd/values.yaml#L834

If you're okay with this I can revert my values file back to the original value of `true`,

Reply:
Yes, lets use `true` as the default. Otherwise it would be a breaking change.

Reply:
My bad... 🫠  Please revert as `true` .